Content.Server/GameTicking/GameTicker.Lobby.cs

```diff
@@ -42,9 +42,16 @@ private string GetInfoText()
                 return string.Empty;
             }
 
+            var mapName = Loc.GetString("game-ticker-no-map-selected");
+            var map = _gameMapManager.GetSelectedMap();
+            if(map != null)
+            {
+                mapName = map.MapName;
+            }
```

Comment:
Maybe make this `mapName = map?.MapName ?? Loc.GetString("game-ticker-no-map-selected");`

Reply:
Done.

Reply:
By the way. Thank you for tips that makes code cleaner. I'm not a C# developer myself.